src/components/Weapons.vue

```diff
@@ -4,9 +4,8 @@
       <h2>{{ title }}s</h2>
       <transition-group name="fade" tag="div" class="weapons">
         <div class="weapon" v-for="weapon in category" :key="weapon.alias">
-          <span class="required-circle" v-if="weapon.required"></span>
           <div class="name" 
-               :class="{ completed: Object.values(weapon.progress).every(Boolean) }" 
+               :class="{ completed: Object.values(weapon.progress).every(Boolean), 'required-weapon': weapon.required }" 
```

Comment:
```suggestion
               :class="{ completed: Object.values(weapon.progress).every(Boolean), 'required': weapon.required }" 
```
I'd change the classname to just 'required', to keep it simple.